app/controllers/application_controller.rb

```diff
@@ -6,6 +6,10 @@ def error_response(object: nil, message: nil, status: nil)
     render json: { message: response_message(object, message) }, status: status
   end
 
+  def unprocessable_response(object)
+    error_response object: object, status: 422
+  end
```

Comment:
@maxwolffe @anthonyhbh since you guys are messing around with controllers, this convenience method can be used to send down error responses for `.save` failures.